roles/matrix-nginx-proxy/tasks/setup_well_known.yml

```diff
@@ -23,3 +23,11 @@
     mode: 0644
     owner: "{{ matrix_user_username }}"
     group: "{{ matrix_user_groupname }}"
+
+- name: Ensure Matrix /.well-known/matrix/support configured
+  template:
+    src: "{{ role_path }}/templates/well-known/matrix-support.j2"
+    dest: "{{ matrix_static_files_base_path }}/.well-known/matrix"
+    mode: 0644
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_groupname }}"
```

Comment:
You probably want to add `when: matrix_homeserver_admin_contacts|length > 0` or similar here and to https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/1931/files#diff-e76cf66accc671f25ab2f2c15659f14425c875b86efe55b6fe37d0e80ca6f006R39.

Reply:
Resolved in [32a604d](https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/1931/commits/32a604d7c16b28dad552c8c87be710bdc9e37d92)